app/budget/services/budgetExcelService.js

```diff
@@ -0,0 +1,456 @@
+				row.push(termDescription);
+				row.push(scheduleCosts[i].subjectCode);
+				row.push(scheduleCosts[i].courseNumber);
+				row.push(scheduleCosts[i].title);
+				row.push(scheduleCosts[i].unitsHigh);
+				row.push(scheduleCosts[i].unitsLow);
+				
+				let sectionGroupCosts = scheduleCosts[i].sectionGroupCosts;
+				for (var _i = 0; _i < sectionGroupCosts.length; _i++) {
+					let childRow = [];
+					childRow.push(sectionGroupCosts[_i].sequencePattern);
+					childRow.push(sectionGroupCosts[_i].enrollment);
+					childRow.push(sectionGroupCosts[_i].sectionCount);
+					childRow.push(sectionGroupCosts[_i].instructorDescription);
+					childRow.push(sectionGroupCosts[_i].originalInstructorDescription);
+					childRow.push(sectionGroupCosts[_i].reason);
+					childRow.push(sectionGroupCosts[_i].taCount);
+					childRow.push(sectionGroupCosts[_i].readerCount);
+					childRow.push(sectionGroupCosts[_i].taCost);
+					childRow.push(sectionGroupCosts[_i].readerCost);
+					childRow.push(sectionGroupCosts[_i].courseCostSubTotal);
+					childRow.push(sectionGroupCosts[_i].instructorCostSubTotal);
+					childRow.push(sectionGroupCosts[_i].totalCost);
+					var parentRow = row.concat(childRow);
+					data.push(parentRow);
+				}
+			}
+		});
+		
+		// Creates Costs worksheet
+		var ws_costs = XLSX.utils.aoa_to_sheet(data); // eslint-disable-line no-undef
+
+		// Set column widths
+		var wscols = [
+			{wch: 15},
+			{wch: 15},
+			{wch: 15},
+			{wch: 30},
+			{wch: 10},
+			{wch: 10},
+			{wch: 10},
+			{wch: 10},
+			{wch: 10},
+			{wch: 25},
+			{wch: 25},
+			{wch: 20},
+			{wch: 10},
+			{wch: 10},
+			{wch: 13},
+			{wch: 13},
+			{wch: 13},
+			{wch: 13},
+			{wch: 13},
+		];
+		ws_costs['!cols'] = wscols;
+
+		/* Add Costs worksheet to workbook */
+		XLSX.utils.book_append_sheet(wb, ws_costs, 'Schedule Costs'); // eslint-disable-line no-undef
+		// Cleans data for the next sheet
+		data.length = 0;
+	
+
+// FUNDS REPORT
+		var lineItems = viewState.calculatedLineItems;
+		// Header
+		data.push(['Type', 'Description', 'Amount']);
+								
+		var row = [];	
+		lineItems.forEach(function(lineItem) {
+			if (lineItem.id > 0 && lineItem.hidden == false){
+				row.push(lineItem.categoryDescription);
+				row.push(lineItem.description);
+				row.push(lineItem.amount);
+				data.push(row);
+				row = [];
+			}
+		});
+		
+		// Creates Funds worksheet
+		var ws_funds = XLSX.utils.aoa_to_sheet(data); // eslint-disable-line no-undef
+
+		// Set column widths
+		var wscols = [
+			{wch: 50},
+			{wch: 50},
+			{wch: 15}
+		];
+		ws_funds['!cols'] = wscols;
+
+		/* add worksheet to workbook */
+		XLSX.utils.book_append_sheet(wb, ws_funds, 'Funds'); // eslint-disable-line no-undef
+		// Cleans data for the next sheet
+		data.length = 0;
+
+
+// INSTRUCTOR SALARIES REPORT
+		var instructors = viewState.calculatedInstructors;
+		data.push(['Instructor', 'Type', 'Cost']);
+
+		var row = [];
+		instructors.forEach(function(instructor){
```

Comment:
Please refactor to follow code style conventions, our current code style has been to put a new line before for loops to separate them.